conversant/prompt_chatbot.py

```diff
@@ -186,11 +188,103 @@ def _update_max_context_examples(
             "tokens"
         )
 
+    def should_stop(self, response:str)-> bool:
+        """Given a response, decides if it ends with a stop sequence and should 
+        be terminated."""
+        for stop_seq in self.client_config["stop_sequences"]: #make into function
+            if response.endswith(stop_seq):
+                return True
+        return False
+    
+    def get_max_reruns(self):
```

Comment:
Missing the return type here

`def get_max_reruns(self) -> int:`

Reply:
Thanks!